tools/benchmark/metrics/hdrvdp.sh

```diff
@@ -4,6 +4,6 @@
 # Use of this source code is governed by a BSD-style
 # license that can be found in the LICENSE file.
 
-./compute_octave_metric.sh "$@" \
+"$(dirname "$0")"/compute_octave_metric.sh "$@" \
```

Comment:
Uh. So we thought of this for the next two lines but not this one?

Reply:
Beats me...